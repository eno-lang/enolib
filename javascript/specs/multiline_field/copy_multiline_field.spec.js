const enolib = require('../..');
const fs = require('fs');
const path = require('path');

const input = `
> Comment

-- Multiline Field
Veni
Vidi
Vici
-- Multiline Field

Copied Multiline Field < Multiline Field
`.trim();

describe('Copying a multiline field', () => {
  test('succeeds', () => {
    const document = enolib.parse(input);
    expect(document.raw()).toMatchSnapshot();
  });
});
