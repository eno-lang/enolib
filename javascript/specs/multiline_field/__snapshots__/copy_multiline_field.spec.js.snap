// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copying a multiline field succeeds 1`] = `
Array [
  Object {
    "Multiline Field": "Veni
Vidi
Vici",
  },
  Object {
    "Copied Multiline Field": "Veni
Vidi
Vici",
  },
]
`;
